NAMESPACE

```diff
@@ -19,6 +20,9 @@ export(subset_wuse)
 export(time_series_data)
 import(DT)
 import(ggplot2)
+import(maptools)
+import(rgeos)
```

Comment:
Seems like I need this library ```rgeos``` else I get an error that ```gpclibPermit()``` is FALSE and the mapping won't work. 
But Checking fails related to ```rgeos```. 
Maybe there is another way to deal with ```gpclibPermit()```? 
I tried installing ```gpclib``` but I get the message _Package is only available in source form, and may need compilation of C/C++/Fortran: ‘gpclib’._ So I haven't followed through.